### ISSUE TITLE
Android Date Picker wrong background color on 6.5.1

### ISSUE THREAD

**Issue by alfonsocj** [react-native-datetimepicker/datetimepicker#678]
# Bug report

## Summary

Hello, we are experiencing an Android bug on 6.5.1 where the date picker's background color changed.
Maybe we need to set it to a default for the picker?

| 6.5.0 | 6.5.1 |
|------| ------|
| <img width="467" alt="4835209c-378d-4ab1-a4b2-cd4fb18e95f4" src="https://user-images.githubusercontent.com/3791120/197590874-714a980c-1356-4cbb-a285-7408a3197d7a.png"> | <img width="408" alt="8bf6d142-0f5f-447c-8a95-ec3ebe9723a9" src="https://user-images.githubusercontent.com/3791120/197590895-12cf40cb-c96d-4ffd-b5a5-3f1fe48b4dc9.png"> |

## Reproducible sample code
I still don't have a reproducible sample outside of our private repo.

## Steps to reproduce

1. …
2. …

Describe what you expected to happen:

1. …
2. …

## Environment info

`npx react-native info` output:

```bash
System:
    OS: macOS 12.6
    CPU: (10) arm64 Apple M1 Pro
    Memory: 90.30 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.16.0 - ~/.nvm/versions/node/v16.16.0/bin/node
    Yarn: 3.2.4 - ~/.yvm/shim/yarn
    npm: 8.11.0 - ~/.nvm/versions/node/v16.16.0/bin/npm
    Watchman: 2022.08.15.00 - /opt/homebrew/bin/watchman
  Managers:
    CocoaPods: 1.11.3 - /Users/fonz/.rbenv/shims/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.4, iOS 16.0, macOS 12.3, tvOS 16.0, watchOS 9.0
    Android SDK:
      API Levels: 28, 29, 30, 31, 32, 33
      Build Tools: 30.0.2, 30.0.3, 31.0.0, 32.0.0, 32.1.0, 33.0.0, 33.0.0
      System Images: android-30 | Google APIs ARM 64 v8a, android-30 | Google APIs Intel x86 Atom, android-30 | Google Play ARM 64 v8a, android-31 | Google APIs ARM 64 v8a, android-32 | Google APIs ARM 64 v8a, android-33 | Google APIs ARM 64 v8a
      Android NDK: 25.0.8775105
  IDEs:
    Android Studio: 2021.3 AI-213.7172.25.2113.9123335
    Xcode: 14.0.1/14A400 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.14.1 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: Not Found
    react-native: Not Found
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

datetimepicker version: 6.5.1

iOS / Android version: Android api 30

**Comment by alfonsocj** [comment]
@vonovak it's related to the `SpinnerDatePickerDialog` style not extending `Theme.AppCompat.Light.Dialog` anymore. If the app's theme sets `android:windowBackground`, the date picker dialog is going to have that background color.

**Comment by vonovak** [comment]
thanks for reporting, give me a few moments to fix, thank you

**Comment by alfonsocj** [comment]
Appreciate it 🙏

**Comment by vonovak** [comment]
@alfonsocj can you please try https://github.com/react-native-datetimepicker/datetimepicker/pull/680/files? Also if you have someone knowledgeable of android styling who could review, that would be nice 👍 
thank you

**Comment by vonovak** [comment]
:tada: This issue has been resolved in version 6.5.3 :tada:

If this package helps you, consider [sponsoring us](https://github.com/sponsors/react-native-datetimepicker)! :rocket:
